lib/qb_integration/invoice.rb

```diff
@@ -1,10 +1,10 @@
 module QBIntegration
   class Invoice < Base
-    attr_accessor :flowlink_invoice
+    attr_accessor :flowlink_invoice, :invoices, :new_page_number
 
     def initialize(message = {}, config)
       super
-      @flowlink_invoice = payload[:invoice]
+      @flowlink_invoice = payload[:invoice] ? payload[:invoice] : {}
```

Comment:
Trinary logic FTW, although it seems a bit redundant. What happens if you do this?
```
@flowlink_invoice = payload[:invoice] || {}
```